hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java

```diff
@@ -25,6 +27,12 @@
     @NotNull
     private ContentType contentType;
 
+    @Min(1024)
+    @Max(2 * 1024 * 1024)
```

Comment:
You implemented limits in two places. The first one is here and the second one is in Configs.java. When you change limits in one place then you have to remember to change them in other one as well. Can we declare them just in one place so that I don't need to remember about changes in other place?